bot/core.py

```diff
@@ -143,6 +149,25 @@ def run():
         bot.load_extension(extension)
     bot.run(token)
 
+#tasks
+
+@tasks.loop(seconds=1800)
+async def reminder():
+    guild = bot.guilds[0]
+    categories = guild.by_category()
+    ctfs = [c for c in guild.categories if c.name != 'Text Channels' and c.name != 'Voice Channels' ]
+    for ctf in ctfs:
+        try:
+            ctf_doc = CTF.objects.get({"name": ctf.name})
+            if ctf_doc.reminder:
+                for c in categories:
```

Comment:
Nomizw dame mporeis na fieis to deutero loop jai na xrisimopiisis apla to `ctf` var epd etsi gireukis mesta categories 2 fores

Reply:
en mporeis  giati to ctf en categories eno to allo eshei mesa ta channels  p exei  to  to ka8e category
.

Reply:
an kamis kati etsi gia na fiei jino to loop? `channel = discord.utils.get(ctf.channels, name='general')`

Reply:
ne ma pio general en na ine? egw gireukw  to general tou specific CTF en kamei specify pio pou oula en na ferei dame

Reply:
resolved